test/helper.rb

```diff
@@ -16,7 +23,58 @@
   add_group 'Backends', 'lib/inspec/backend'
 end
 
-require 'minitest/autorun'
+##
```

Comment:
Wizardry (in my eyes) with supporting documentation. So so so good.

Reply:
Wow.